pages/1.13/security/ent/users-groups/add-external-user/index.md

```diff
@@ -62,7 +62,7 @@ Group size is limited to 100 users. To increase this limit, contact Mesosphere c
 
 6. When completed, your dialog should look something like the following.
```

Comment:
Rephrase `When completed, your dialog should look something like the following.`
with `When completed, the LDAP group import configuration dialog appears as shown in Figure 1.`